README.md

```diff
@@ -22,3 +22,6 @@ The typical image workflow comprises of the following steps:
 - How to convert RHEL qcow2 to ova image format - [guide](docs/RHEL%20Qcow2%20to%20OVA.md)
 - How to convert RHCOS(Red Hat CoreOS) qcow2 to ova image format - [guide](docs/RHCOS%20Qcow2%20to%20OVA.md)
 - Advanced scenarios for Qcow2 to ova image conversion - [guide](docs/Advanced%20Scenarios%20for%20Qcow2%20to%20OVA.md)
+
+### Samples and Examples
+For sample examples look at [sample](samples/Sample%20Examples.md) folder
```

Comment:
Please take a look at the [samples](samples/README.md) folder for end-to-end examples.

Reply:
Done